README.md

```diff
@@ -14,6 +14,7 @@ Requirements
 1. A TrueNAS system with at least 8 Gb of memory and a minimum 20 GiB disk.  Suggested version >= 12.x to use API v2.0
 2. Another system running either DevStack (Train or higher) or an OpenStack storage node.
 3. This driver is now upgraded to Python 3, so is usable by OpenStack versions from Train on.  
+4. This driver should be running on a single cinder driver instance deployment, High Available (Active-Active) multiple cinder driver instance deployment is not supported.
```

Comment:
IX cinder driver does not implement the necessary lock for transaction across multiple instance as for now, hence when running in multiple High Available deployment will result unexpected behaviors and lead to user data corruptions. Detail information here: https://docs.openstack.org/cinder/latest/contributor/high_availability.html